src/openrct2/localisation/StringIds.h

```diff
@@ -3896,6 +3893,16 @@ enum
 
     STR_VIEWPORT_TRANSPARENT_WATER = 6440,
 
+    STR_AT_LEAST_ONE_FOOTPATH_NON_QUEUE_SURFACE_OBJECT_MUST_BE_SELECTED = 6441,
+    STR_AT_LEAST_ONE_FOOTPATH_QUEUE_SURFACE_OBJECT_MUST_BE_SELECTED = 6442,
+    STR_AT_LEAST_ONE_FOOTPATH_RAILING_OBJECT_MUST_BE_SELECTED = 6443,
+    STR_OBJECT_SELECTION_FOOTPATH_SURFACES = 6444,
+    STR_OBJECT_SELECTION_FOOTPATH_RAILINGS = 6445,
+    STR_TILE_INSPECTOR_FOOTPATH_SURFACE_NAME = 6446,
+    STR_TILE_INSPECTOR_FOOTPATH_RAILINGS_NAME = 6447,
+
+    STR_UNSUPPORTED_OBJECT_FORMAT = 6448,
+
```

Comment:
STR_AT_LEAST_ONE_PATH_OBJECT_MUST_BE_SELECTED is no longer used and should be removed as well.